microsoft/electionguard-admin-device
# Feature/ballot reg empty files

### Issue

Fixes #53

### Description
allows bypassing loading any files if they do not exist

Should be merged after PR #59 

### Testing
1. announce trustees
2. try to load a ballot file that does not exist (expected: turns yellow)
3. repeat for cast/spoil
4. hit next

Expected: it allows you to go to the tally page

### Checklist
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.


💔Thank you!
